proto/Makefile

```diff
@@ -42,8 +42,8 @@ build/proto.stamp: $(source_files)
 build/swagger:
 	mkdir -p build/swagger
 
-$(swagger_out): $(swagger_in) build/swagger
-	protoc -I src $< --swagger_out=logtostderr=true:build/swagger
+$(swagger_out): $(source_files) build/swagger
```

Comment:
@rb-determined-ai should look at this not me.

Reply:
I this I should also update the comment on line 10

Reply:
yes